app/src/main/res/color/bottom_navigation_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_checked="true" android:color="@color/colorAccent" />
+    <item android:color="@color/dark_grey"  />
```

Comment:
The default colour of the bottom navigation bar has changed

Reply:
updated
![screenshot_1533900498](https://user-images.githubusercontent.com/16715499/43955625-a98a1bc0-9cbe-11e8-903e-bcc82640145e.png)